drivers/storage/COMPONENT_NUSD.lib

```diff
@@ -0,0 +1 @@
+https://github.com/OpenNuvoton/NuMaker-mbed-SD-driver/#f9f81fcba836a651e57fa3b3ed31ba5954142306
```

Comment:
@ccli8, @SeppoTakalo I'm a bit skeptical about creating dependencies to external repositories.

Reply:
That's inevitable because some boards like NuMaker-PFM-NUC472 and NuMaker-IoT-M263A just support `NUSD`, no other external storage.